api/onnx_web/chain/tile.py

```diff
@@ -207,17 +229,11 @@ def process_tile_spiral(
     overlap: float = 0.5,
     **kwargs,
 ) -> Image.Image:
-    if scale != 1:
-        raise ValueError("unsupported scale")
-
     width, height = kwargs.get("size", source.size if source else None)
 
     # spiral uses the previous run and needs a scratch texture for 3x memory
-    image = Image.new("RGB", (width * scale, height * scale))
-    image.paste(source, (0, 0, width, height))
 
     tiles: List[Tuple[int, int, Image.Image]] = []
-    tiles.append((0, 0, source))
```

Comment:
Is this `(0, 0)` tile added by the first iteration of the loop now?

Reply:
The (0,0) tile added here didn't make a lot of sense to me, especially because there's no size checking for it. I believe this (0,0) tile was related to the old partial outpainting idea we were talking about. Either way, yes, the correctly tiled (0,0) tile is created in generate_tile_spiral